util/release/main.go

```diff
@@ -0,0 +1,222 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"sync"
+)
+
+func main() {
+	setup()
+	stageProtos()
+	parallelize(compileDescriptors, tarProtos, compileBinaries)
+	teardown()
+}
+
+func setup() {
+	distDir := filepath.Join(showcaseDir(), "dist")
+	os.RemoveAll(distDir)
+	if err := os.MkdirAll(distDir, 0755); err != nil {
+		log.Fatalf("Failed to make the directory %s: %v", distDir, err)
+	}
+
+	tmpDir := filepath.Join(showcaseDir(), "tmp")
+	os.RemoveAll(tmpDir)
```

Comment:
I think we should check errs here as well.

Reply:
Done.